VALUES_AND_PRIORITIZAION.md

```diff
@@ -0,0 +1,65 @@
+# Values and Priorities
+
+> Document Status: Proposal, Work In Progress
+
+## Context
+
+Define the `Technical` values shared by the project and their relatively priority
+that are applied when making tradeoffs.
+
+
+## Values and priority level
+
+This is only a subset of the overall prioties, listing only those which are at the
+top of the priority list. Anything on this list is very important for the project
+despite the relative priorities shown.
+
+- Prio 1 - Developer experience
+- Prio 2 - Stability
+- Prio 3 - Operational qualities
+- Prio 4 - Node.js maintainer experience
+- Prio 5 - Technology and API Currency
+
+## Value descriptions
+
+### Developer Experience
+We value ensuring that developers are productive and enjoy developing with Node.js. Some key elements of this include:
+  - Approach-ability (both technical and community)
+  - Great Documentation 
+  - Bundling API/components which reduce the friction of using them, even though they could be external
+  - Enabling/supporting external packages to ensure overall developer experience
```

Comment:
It's kind of unclear what is meant by this

Reply:
What we were trying to say was that developer experience for the Node.js ecosystem is not only about the APIs provided directly by Node.js itself but also those provided by external packages as well.  In that context it is important to provide APIs, or whatever else is needed to support external packages in helping the overall developer experience.

Any suggestions on how to better capture that that they short version we had?

Reply:
maybe something like "Enable better developer experiences in packages and applications built upon Node.js"?